lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -86,14 +86,15 @@ export class ItinerarySummary extends Component {
   }
 
   render () {
-    const { currency, itinerary } = this.props
+    const { currency, defaultFare, itinerary } = this.props
     const { LegIcon } = this.context
 
     const {
       maxTNCFare,
       minTNCFare,
-      transitFare
-    } = coreUtils.itinerary.calculateFares(itinerary)
+      transitFares
+    } = coreUtils.itinerary.calculateFares(itinerary, true)
+    const transitFare = (transitFares?.[defaultFare] ?? transitFares.regular).transitFare
```

Comment:
Is optional chaining here necessary since `transitFares.regular` is referenced directly after?

Reply:
Good catch! Got myself confused here while bug fixing. Have now replaced this with a more standard `||`. It's for situations where `defaultFare` isn't `regular`

Reply:
Addressed in 88c0af89a48df3baed5064b7a4dd86720c3ca06d